core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/creation/ArraysAsListUnitTest.java

```diff
@@ -0,0 +1,25 @@
+package com.baeldung.list.creation;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ArraysAsListUnitTest {
+
+@Test
+void shouldThrowUnsupportedOperationExceptionWhenAddingElementsToFixedList() {
+    List<Integer> list = Arrays.asList(1, 2, 3, 4, 5);
+    assertThrows(UnsupportedOperationException.class, () -> list.add(6));
+}
+
+@Test
+void shouldThrowUnsupportedOperationExceptionWhenRemovingElementsFromFixedList() {
+    List<Integer> list = Arrays.asList(1, 2, 3, 4, 5);
+    assertThrows(UnsupportedOperationException.class, () -> list.remove(1));
+}
+
```

Comment:
I'd expect to see unit tests for the contents of the list in the positive case, as well as its exception throwing